include/rabit/internal/utils.h

```diff
@@ -66,10 +66,19 @@ const int kPrintBuffer = 1 << 12;
 /*! \brief we may want to keep the process alive when there are multiple workers
  * co-locate in the same process */
 extern bool STOP_PROCESS_ON_ERROR;
+  
```

Comment:
include/rabit/internal/utils.h:69:  Line ends in whitespace.  Consider deleting these extra spaces.  [whitespace/end_of_line] [4]